src/amberscript/parser.cc

```diff
@@ -732,8 +732,22 @@ Result Parser::ParsePipelineBind(Pipeline* pipeline) {
         token = tokenizer_->NextToken();
         if (!token->IsInteger())
           return Result("invalid value for BIND LOCATION");
+        auto location = token->AsUint32();
 
-        r = pipeline->AddColorAttachment(buffer, token->AsUint32());
+        uint32_t base_mip_level = 0;
+        token = tokenizer_->PeekNextToken();
+        if (token->IsString() && token->AsString() == "BASE_MIP_LEVEL") {
```

Comment:
Can you update the docs/amber_script.md file so I can see the places where MIP_LEVEL is used? Why call it BASE_MIP_LEVEL instead of just MIP_LEVEL here? Does base have a specific meaning?

Reply:
Sure. MIP_LEVEL determines the number of mip levels the texture uses. Base mip level works as an offset. Using this offset we can render to a specific mip level when bound as a frame buffer, or read a certain mip level without LOD function when sampling a texture.